php/post-types/class-block-post.php

```diff
@@ -81,40 +81,57 @@ public function register_hooks() {
 	 * @return void
 	 */
 	public function register_controls() {
-		$controls = array(
-			'text'        => new Controls\Text(),
-			'textarea'    => new Controls\Textarea(),
-			'url'         => new Controls\URL(),
-			'email'       => new Controls\Email(),
-			'number'      => new Controls\Number(),
-			'color'       => new Controls\Color(),
-			'image'       => new Controls\Image(),
-			'select'      => new Controls\Select(),
-			'multiselect' => new Controls\Multiselect(),
-			'toggle'      => new Controls\Toggle(),
-			'range'       => new Controls\Range(),
-			'checkbox'    => new Controls\Checkbox(),
-			'radio'       => new Controls\Radio(),
+		$control_names = array(
```

Comment:
This could have been a property, but I don't think it's needed outside this function. Also, it might add to confusion, as there are `$controls` and `$pro_controls` properties.